static/js/services/send-message.js

```diff
@@ -130,15 +131,15 @@ var _ = require('underscore'),
         });
       };
 
-      var createTask = function(settings, recipient, message, user, uuid) {
+      var createTask = function(settings, recipient, message, user) {
         var task = {
           messages: [{
             from: user && user.phone,
             sent_by: user && user.name || 'unknown',
             to: libphonenumber.normalize(settings, recipient.phone) || recipient.phone,
             contact: recipient.contact,
             message: message,
-            uuid: uuid
+            uuid: uuid.v4()
```

Comment:
I don't think the decision of which version to be used should be made here. Also this means that `require()` will include more code than `require('uuid/v4')`